platform/darwin/src/MGLTileSource.h

```diff
@@ -161,14 +161,14 @@ extern MGL_EXPORT const MGLTileSourceOption MGLTileSourceOptionDEMEncoding;
 typedef NS_ENUM(NSUInteger, MGLDEMEncoding) {
 
     /**
-     Raster tiles generated with the Mapbox encoding formula where
-     elevaion = -10000 + ((R * 256 * 256 + G * 256 + B) * 0.1)
+     Raster tiles generated with the Mapbox encoding formula. 
+     See more details here: https://www.mapbox.com/help/access-elevation-data/#mapbox-terrain-rgb 
```

Comment:
[Documentation comments use Markdown format](https://github.com/mapbox/mapbox-gl-native/blob/master/platform/ios/DEVELOPING.md#making-any-symbol-public). Link “Mapbox encoding formula” to this URL using Markdown format. (Technically, HTML syntax would be preferable, but Quick Help’s formatting is currently broken, so the only parser we care much about right now is jazzy’s.)